docs/network_firewall.rst

```diff
@@ -465,20 +469,38 @@ to add a rule.
 Once you've set up the firewall, exit the Unsafe Browser, and continue
 with the "Keeping pfSense up to date" section below.
 
-Configuration Reference Templates
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+SecureDrop Configuration (Templates)
+------------------------------------
+
+Alternatively, you can load the provided ``.xml`` templates.
+
+First navigate to **Diagnostics ▸ Backup & Restore**:
 
 As an alternative to the provided screenshots, you can examine the provided
-``.xml`` templates as a reference:
+``.xml`` templates as a reference. However, because of the Tails chroot,
+you can not download the ``.xml`` files directly. Using the Unsecure Browser,
+download the files from `GitHub <https://github.com/freedomofpress/securedrop/tree/develop/install_files/network_firewall/>`__
+as "Raw" and save them as plaintext documents.
+
+|Backup & Restore|
```

Comment:
This "Backup and Restore" screenshot should be right after "First navigate to **Diagnostics ▸ Backup & Restore**:" right?